tests/backend/base/HttpTestCase.php

```diff
@@ -10,19 +10,29 @@
 
 namespace Tests\Base;
 
+use demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure;
+use demosplan\DemosPlanCoreBundle\Entity\User\User;
 use demosplan\DemosPlanCoreBundle\EventListener\SetHttpTestPermissionsListener;
+use Lexik\Bundle\JWTAuthenticationBundle\Security\Authentication\Token\JWTUserToken;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
 use Symfony\Bundle\FrameworkBundle\KernelBrowser;
 
 class HttpTestCase extends FunctionalTestCase
 {
+    public const JWT_AUTHORIZATION_HEADER = 'HTTP_X-JWT-Authorization';
```

Comment:
The header is named `X-JWT-Authorization`  and you should rather define a new constant in `demosplan\DemosPlanCoreBundle\Application\Header` than redefine it here